evalml/tests/model_understanding_tests/test_feature_explanations.py

```diff
@@ -0,0 +1,356 @@
+    assert expected in out
+    caplog.clear()
+
+    # Objective is None, target is None
+    readable_explanation(pipeline, importance_method="feature")
+    expected = "The output is"
+    out = caplog.text
+    assert expected in out
+    caplog.clear()
+
+
+@patch("evalml.pipelines.PipelineBase.feature_importance", new_callable=PropertyMock)
+def test_readable_explanation_somewhat_important_features(
+    mock_feature_importance, caplog
+):
+    pipeline = BinaryClassificationPipeline(elasticnet_component_graph)
+    pipeline._is_fitted = True
+    mock_feature_importance.return_value = pd.DataFrame(
+        {
+            "feature": ["heavy 1", "somewhat 1", "somewhat 2", "somewhat 3"],
+            "importance": [0.55, 0.17, 0.15, 0.13],
+        }
+    )
+
+    readable_explanation(pipeline, importance_method="feature")
+    expected_somewhat = (
+        "is somewhat influenced by somewhat 1, somewhat 2, and somewhat 3."
+    )
+    out = caplog.text
+    assert expected_somewhat in out
+    caplog.clear()
+
+    mock_feature_importance.return_value = pd.DataFrame(
+        {
+            "feature": ["heavy 1", "heavy 2", "heavy 3", "somewhat 1"],
+            "importance": [0.32, 0.3, 0.23, 0.15],
+        }
+    )
+    readable_explanation(pipeline, importance_method="feature")
+    expected_somewhat = "is somewhat influenced by somewhat 1."
+    out = caplog.text
+    assert expected_somewhat in out
+    caplog.clear()
+
+
+@patch(
+    "evalml.model_understanding.feature_explanations.calculate_permutation_importance"
+)
+def test_readable_explanation_detrimental_features(
+    mock_permutation_importance, caplog, fraud_100
+):
+    pipeline = BinaryClassificationPipeline(elasticnet_component_graph)
+    X, y = fraud_100
+    pipeline._is_fitted = True
+    mock_permutation_importance.return_value = pd.DataFrame(
+        {
+            "feature": [
+                "heavy 1",
+                "somewhat 1",
+                "somewhat 2",
+                "somewhat 3",
+                "detrimental 1",
+            ],
+            "importance": [0.55, 0.17, 0.15, 0.13, -0.1],
+        }
+    )
+    readable_explanation(pipeline, X, y)
+    expected_detrimental = "The feature detrimental 1 detracted from model performance. We suggest removing this feature"
+    out = caplog.text
+    assert expected_detrimental in out
+    caplog.clear()
+
+    mock_permutation_importance.return_value = pd.DataFrame(
+        {
+            "feature": ["heavy 1", "somewhat 1", "detrimental 1", "detrimental 2"],
+            "importance": [0.55, 0.13, -0.1, -0.25],
+        }
+    )
+    readable_explanation(pipeline, X, y)
+    expected_detrimental = "The features detrimental 1 and detrimental 2 detracted from model performance. We suggest removing these features"
+    out = caplog.text
+    assert expected_detrimental in out
+    caplog.clear()
+
+
+@patch("evalml.pipelines.PipelineBase.feature_importance", new_callable=PropertyMock)
```

Comment:
Can we add a test where:
- The importance of one or more of the elements is equal to the threshold for 'heavy'